lexicons/com/atproto/server/createSession.json

```diff
@@ -33,7 +33,12 @@
             "didDoc": { "type": "unknown" },
             "email": { "type": "string" },
             "emailConfirmed": { "type": "boolean" },
-            "emailAuthFactor": { "type": "boolean" }
+            "emailAuthFactor": { "type": "boolean" },
+            "status": {
+              "type": "string",
+              "description": "Hosting status of the account. If not specified, then assume 'active'.",
+              "knownValues": ["active", "takendown", "suspended", "deactivated"]
```

Comment:
How do we feel about the fact that `active` may appear here but is not a hosting status under `com.atproto.sync`?

I think I like the convention that a status indicates a reason for a no-host, so if you see a value you don't understand you can at least assume it still indicates no-host.  One result of that is that the logic around this field could be `const noHost = !!status && status !== 'active'`, which is pretty nice.  Would it be even nicer if it were just `const noHost = !!status`?  I notice in a fair amount of logic in this PR we check explicitly for the known values to determine a no-host.

Reply:
ended up updating this to match the sematnics of `getRepoStatus` with the `active` boolean